lighthouse-core/audits/byte-efficiency/library-suggestions.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
this is handwritten, right? if so, we prefer `.js` files so we can leave comments as needed.